client/verta/tests/unit_tests/test_runtime.py

```diff
@@ -22,6 +22,8 @@ def clean_thread():
     yield
     if hasattr(runtime._THREAD, 'logs'):
         runtime._THREAD.__delattr__('logs')  # clean-up any existing logs.
+    if hasattr(runtime._THREAD, 'validate'):
+        runtime._THREAD.__delattr__('validate')  # clean-up any existing flag.
```

Comment:
Like I mentioned in [my other comment](https://github.com/VertaAI/modeldb/pull/3500#discussion_r1089220329), I'd recommend sticking with `delattr(runtime._THREAD, 'logs')` rather than directly calling a dunder method.